lib/Tumblr/StreamBuilder/FencepostRanking/CacheFencepostProvider.php

```diff
@@ -37,13 +37,19 @@ final class CacheFencepostProvider extends FencepostProvider
      */
     private $cache;
 
+    /**
+     * @var int TTL in seconds. Defaults to 1 week.
+     */
+    private $ttl_seconds;
```

Comment:
```suggestion
    private int $ttl_seconds;
```

i think we can, and should, use typehints when possible for new code